src/common/k8s-api/kube-api.ts

```diff
@@ -173,7 +173,6 @@ export class KubeApi<T extends KubeObject> {
     this.request = request;
     this.objectConstructor = objectConstructor;
 
-    this.checkPreferredVersion();
```

Comment:
we don't need this anymore?

Reply:
I asked the same question on Slack few days ago. Everything seems to work without -> calling this in a constructor has serious side-effects because it will trigger api requests when api is instantiated ... at that point it might not even have a context where it should make the api call. Feels like a wrong place for this call (maybe it's some legacy?).

Reply:
yes, agree